templates/design_system/new_authorize_v2.html

```diff
@@ -80,8 +81,9 @@ <h1 class="ds-text-heading--2xl ds-u-margin-bottom--3">Connect your Medicare cla
                                     type="checkbox"
                                     value="True"
                                     checked=""
+                                    aria-describedby="allow_access block_access"
```

Comment:
Can we put a period after each dynamic message here? We don't need a period for the dynamic message you can see, but when we put them back to back like this, there should be a period after each one.

Reply:
Done.